staging/jenkins/Chart.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: v1
+name: jenkins
+home: https://jenkins.io/
+version: 1.7.3
```

is this the convention we are using in mesosphere charts, ie our charts have the same version as the upstream version?

---

No convention: we should increase the patch version here for our changes:
```suggestion
version: 1.7.4
```

---

we have no changes on the base tech, Shouldnt we keep the versions the same?

---

:thinking: CI is hardcoded to require the chart version be updated as far as upstream we'll just basically be "diverged" until we merge something up there.